docs/developer/developer.adoc

```diff
@@ -46,7 +46,7 @@ On RHEL / Fedora / CentOS, the command to install the build dependencies is:
 ----
 sudo yum install \
 cmake dbus-devel GConf2-devel libacl-devel libblkid-devel libcap-devel libcurl-devel \
-libgcrypt-devel libselinux-devel libxml2-devel libxslt-devel make openldap-devel \
```

Comment:
`attr/xattr.h` does not exist anymore in Fedora 28 and newer. Instead, we should use `sys/xattr.h`. The file `sys/xattr.h` is provided by `glibc-headers` RPM package. 
Please compare F27: https://koji.fedoraproject.org/koji/rpminfo?rpmID=10561854 with F28: https://koji.fedoraproject.org/koji/rpminfo?rpmID=14495196 . Notice that `/usr/include/attr/xattr.h` is not present  anymore in  `libattr-devel` RPM package in F28.